MycroftAI/mycroft-core
# Refactor/skill settings split

## Description
Complete rethink of how we deal with settings and settingsmeta in core

## How to test
Make changes to settingsmeta.json for a skill.  Confirm that the changes are accurately reflected on the backend.  Make changes to skill settings at account.mycroft.ai.  Confirm that changes are accurately reflected in settings.json

## Contributor license agreement signed?
CLA [yes]
